### ISSUE TITLE
bidirect_enc_dynam_dec.py config file

### ISSUE THREAD

**Issue by mcmillanmajora** [Linguistics575/POCRE#4]
A lot of the parameters of model training, such as the number of training iterations, the number of additional edits permitted in the output, the batch size, the state size, the embedding size, the learning rate, and the location of the saved model, are currently hardcoded in the file. Future work will include making this information an argument in the model call, perhaps as a configuration file.
